admin_guide/integrations.rst

```diff
@@ -0,0 +1,157 @@
+External Integrations
```

Comment:
@samay-sharma @begriffs Is there something more descriptive we could say for the Title?